includes/class-sensei-templates.php

```diff
@@ -148,6 +148,10 @@ public static function template_loader( $template = '' ) {
 		$find = array( 'sensei.php' );
 		$file = '';
 
+		if ( Sensei()->feature_flags->is_enabled( 'optional_templates' ) && ! apply_filters( 'sensei_use_sensei_template', true ) ) {
```

Comment:
I'm wondering if we really need the `sensei_use_sensei_template` filter, or if we could call directly a function like the `skip_single_course_template` with the checks, including the feature flag check. WDYT?

Reply:
I've added this in a way that we can easily move it to production by removing the feature flag check. Also this is less complicated and more flexible than coupling in the outline block here.